backend/config/passport.js

```diff
@@ -22,7 +22,12 @@ export const samlStrategy = new saml.Strategy({
   // Identity Provider's public key
   cert: samlConfig.cert,
   validateInResponseTo: false,
-  disableRequestedAuthnContext: true
+  disableRequestedAuthnContext: true,
+
+  //forces user to reauthenticate every time they log on
+  // TODO for testing purposes only, remember to remove
+  forceAuthn: true
```

Comment:
Do we want to make a separate PR for this? so that it's got a heavier paper trail?

Reply:
I feel like the TODO should be enough just to remember we did it. Idk what everyone else thinks

Reply:
Yeah I feel like this could be something really easy to lose track of down the line. I wouldn't think necessarily a pull request. Maybe have it in writing as an issue to address later on!